CandidateExercise/ConfigurationExercise/Config.cs

```diff
@@ -88,22 +88,60 @@ public void Shutdown()
 
         public ConfigValue GetValue(string key)
         {
-            throw new NotImplementedException();
+            // Creating a new List and adding the key in the list
+            IList<string> keyList = new List<string>(); keyList.Add(key);
```

Comment:
Please move the second operation to a new line. That way there is less chance of a future developer missing or deleting  this call when attempting to manipulate the first operation.